src/main/scala/io/smartdatalake/definitions/Environment.scala

```diff
@@ -30,6 +30,24 @@ object Environment {
    */
   var hdfsAclsRequired: Boolean = false
 
+  /**
+   * Modifying ACL's is only allowed up to the following level (default=2)
+   * See also [[io.smartdatalake.util.misc.AclUtil]]
+   */
+  var hdfsAclsMinLevelPermissionModify = 2 // up to user home
+
+  /**
+   * Overwriting ACL's is only allowed up to the following level (default=2)
```

Comment:
```suggestion
   * Overwriting ACL's is only allowed up to the following level (default=5)
```
I assume our Environment reflects the "default".